mesonbuild/backend/ninjabackend.py

```diff
@@ -112,13 +123,17 @@ def write(self, outfile):
                 if not should_quote or i == '&&': # Hackety hack hack
                     quoter = ninja_quote
                 else:
-                    quoter = lambda x: ninja_quote(quote_func(x))
+                    quoter = lambda x: ninja_quote(qf(x))
```

Comment:
__[flake8]__
* [E731] do not assign a lambda expression, use a def

<!-- Comment by [SideCI](https://sideci.com) -->